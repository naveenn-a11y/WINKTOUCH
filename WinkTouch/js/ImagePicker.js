/**
 * @flow
 */
'use strict';

import React, { Component } from 'react';
import { Text } from 'react-native';

export class ImagePicker extends Component {
  render() {
    return <Text>Image android picker</Text>
  }
}