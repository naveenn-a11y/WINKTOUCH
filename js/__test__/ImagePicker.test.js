import renderer from 'react-test-renderer'
import { ImagePicker } from '../ImagePicker'

it('renders correctly', () => {
  const tree = renderer.create(<ImagePicker />).toJSON()
  expect(tree).toMatchSnapshot()
})