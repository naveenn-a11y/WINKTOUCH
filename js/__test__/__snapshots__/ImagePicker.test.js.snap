// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
>
  <View>
    <Image
      source={
        Object {
          "testUri": "../../../js/image/camera.png",
        }
      }
      style={
        Object {
          "height": 14.6484375,
          "padding": 14.6484375,
          "resizeMode": "contain",
          "width": 14.6484375,
        }
      }
    />
    <Image
      source={null}
      style={
        Object {
          "height": 190.4296875,
          "resizeMode": "contain",
          "width": 256.34765625,
        }
      }
    />
  </View>
</View>
`;
